incubator/kubeless/templates/cluster-role.yaml

```diff
@@ -51,6 +51,7 @@ rules:
   - functions
   - httptriggers
   - cronjobtriggers
+  {{ if .Values.kafkaTrigger.enabled }}- kafkatriggers {{ end }}
```

Comment:
we should have a different clusterRole for this, the Kafka controller only needs access to:
```
apiVersion: rbac.authorization.k8s.io/v1beta1
kind: ClusterRole
metadata:
  name: kafka-controller-deployer
rules:
- apiGroups:
  - ""
  resources:
  - services
  - configmaps
  verbs:
  - get
  - list
- apiGroups:
  - kubeless.io
  resources:
  - functions
  - kafkatriggers
  verbs:
  - get
  - list
  - watch
  - update
  - delete
```